Synergism.css

```diff
@@ -2597,6 +2606,10 @@ p[id$="BlessingsTotal"] {
     min-width: 1212px;
 }
 
+#historySingularity .historyTableWrap {
```

Comment:
re-use the rule as mentioned above

Reply:
Do you mean the dot-notation rule so change it to #historySingularity.historyTableWrap {
Or is there another rule I'm not noticing?

Reply:
He means stuff like this
![image](https://user-images.githubusercontent.com/107661324/178029283-81dfdd1a-a018-467d-8579-895472a3c9fc.png)
Can be done like this
#historySingularity,
#historyAscend {
    margin: 0 auto;
}